caramel/config.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python
```

Comment:
This file should not have a shebang as it should not be executable on it's own.

Reply:
All .py files in the project have them, so was only following the precedent.